### PR TITLE
Tickets/infra 227

### PR DESCRIPTION
**Pull request by yukimoritani**

### CODE REVIEW

**Review by naoyukitamura — CHANGES_REQUESTED** [review]
Requirements and recommendations to summary and description when a ticket is filed are well written. As well as these, comments to be given when a review is requested and a ticket is closed need to be equally informative (e.g. how test is done, what result(s) is output and why it is satisfactory). So it would be important to write requirements/recommendations somewhere in this procedure. 

Apart from it, I would agree this document is well shaped.

File: development-management/developement_practice.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
+PFS Software Developement Practice
```
Comment (hassanxp) [inline development-management/developement_practice.rst]: Developement -> Development (throughout document and in filename)

File: development-management/developement_practice.rst [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+
+System-level (L2) requirements relating with PFS software developement
+----------------------------------------------------------------------
+
+(Note that there are L3 and L4 requirements for software packages.)
+
+Below listed are a couple of the most important requirements for software developments.
+
+Coding Standars
```
Comment (hassanxp) [inline development-management/developement_practice.rst]: Standars -> Standards

File: development-management/developement_practice.rst [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+^^^^^^^^^^^^^^^
+
+`RES-SYS-1103 <https://sumire.pbworks.com/w/page/76623143/REQ%20SYS%201103>`_ describes the requirement of the comutar language used for the PFS project:
```
Comment (hassanxp) [inline development-management/developement_practice.rst]: comutar -> computer (or better 'software'?)

File: development-management/developement_practice.rst [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+    Source codes of PFS software modules shall be written in major languages like those considered as industry standard.
+
+Documentation
+^^^^^^^^^^^^^
+
+`REQ SYS 458 <https://sumire.pbworks.com/w/page/76623227/REQ%20SYS%20458>`_ describes the requirement of documentations:
```
Comment (hassanxp) [inline development-management/developement_practice.rst]: 'requirement of documentations' -> 'documentation requirements'

**Review by hassanxp — APPROVED** [review]
Minor typos spotted, if you have time to fix. Otherwise looks good.

**Review by naoyukitamura — APPROVED** [review]

File: development-management/development_practice.rst [hunk 5]
```diff
@@ -0,0 +1,115 @@
+PFS Software Development Practice
```
Comment (hassanxp) [inline development-management/development_practice.rst]: Practice->Practices, as this document covers a number of practices that developers are encouraged to follow. The filename should also be updated accordingly.

**Pull request merged**